admin_manual/installation/linux_installation.rst

```diff
@@ -3,27 +3,70 @@ Preferred Installation Method
 =============================
 
 For production environments, we recommend the installation from the tar archive. 
-This applies in particular to scenarios, where the web server, storage and database are on separate machines. 
+This applies in particular to scenarios, where the Web server, storage and database are on separate machines. 
+In this constellation, all dependencies and requirements are managed by the package management 
+of your operating system, while the ownCloud code itself is maintained in a sequence of simple steps 
+as documented in our instructions for the :doc:`Manual Installation on Linux <source_installation>` and the :doc:`Manual ownCloud Upgrade <../maintenance/manual_upgrade>`.
 
-In this constellation, all dependencies and requirements are managed by your operating system's package manager, while the ownCloud code itself is maintained in a sequence of simple steps as documented in our instructions for the :doc:`Manual Installation on Linux <source_installation>` and the :doc:`Manual ownCloud Upgrade <../maintenance/manual_upgrade>`.
 The package installation is for single-server setups only.
 
-Repositories
-------------
+Changes in the Linux Distribution Packages 9
+--------------------------------------------
+
+Linux distribution packages (from `Open Build Service`_) have been divided into 
+multiple packages for ownCloud 9: ``owncloud``, ``owncloud-deps`` and ``owncloud-files``. 
```

Comment:
Not sure if we should still speak about version 9 here. Additionally it is decided that - when using our repos - the only recommended package to use is 'owncloud-files'. The others are still built but not recommended.
You could add some input from my comment https://github.com/owncloud/documentation/issues/3210#issuecomment-314465622 to explain why this change was done.